sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -380,16 +425,19 @@ object ViewHelper {
     // for createViewCommand queryOutput may be different from fieldNames
     val queryOutput = analyzedPlan.schema.fieldNames
 
+    val conf = SQLConf.get
```

Comment:
since we have `session` passed in, seems better to use `session.sessionState.conf`

Reply:
done